src/Illuminate/Validation/ValidationRuleParser.php

```diff
@@ -250,7 +250,7 @@ protected static function parseParameters($rule, $parameter)
     {
         $rule = strtolower($rule);
 
```

Comment:
or just `return preg_match('/(not(_)?)?regex/i', $rule) ? [$parameter] : str_getcsv($parameter);` ?